uwblueprint/extend-a-family
# Check if user has authenticated themselves thru email

added a check that verifies that the user is authenticated before allowing them onto home page. User must go into their email now and click the verification link before being allowed onto the app. 

## Notion ticket link

[User Verification](https://www.notion.so/uwblueprintexecs/User-Verification-f4620f59ea49484d9f342d788bcce82b?pvs=4)



## Implementation description
* login and sign up pages calls backend api to check if user has verified themselves thru email. 



## Steps to test
1. Do a test where user signs up and attempts to log in without verifying themselves
2. then do a test where they actually log in



## What should reviewers focus on?
* 


## Checklist
